app/src/test/java/com/example/benne/daisyapp2/NCCParserUnitTests.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.benne.daisyapp2
+
+import com.example.benne.daisyapp2.data.daisy202.DaisyBook
+import com.example.benne.daisyapp2.data.daisy202.NCCParser.parseNCC
+import org.junit.Assert.assertEquals
+import org.junit.BeforeClass
+import org.junit.Test
+
+import java.io.File
+import java.io.FileNotFoundException
+
+class NCCParserUnitTests {
+    companion object {
+        private var book: DaisyBook? = null
+        private val path = File("").absolutePath + "\\src\\test\\java\\com\\example\\benne\\daisyapp2"
```

Comment:
this test is failing. You might need to use `Path.Join` or something so that it works on linux / mac